docs/dev/development.md

```diff
@@ -120,17 +120,22 @@ See the previous section for instructions.
     pytest .
     ```
 
-4. (**OPTIONAL**) include your development copy of cirq in your python path.
+4. (**OPTIONAL**) include your development copy of cirq and its subpackages in your python path.
 
     ```bash
-    PYTHONPATH="$(pwd)":"${PYTHONPATH}"
+    ./dev_tools/pypath
     ```
     
-    or add it to the python path, but only in the virtualenv.
+    or add it to the python path, but only in the virtualenv by first listing the modules
     
     ```bash
-    add2virtualenv ./
+    python dev_tools/modules.py list --mode package-path > 
     ```
+    and then adding these to the virtualenv:
+    ```bash
+    add2virtualenv <paste modules from last command>
+    ```
+    (Typically `add2virtualenv` is not executable using xargs, so this two step process is necessary.)
```

Comment:
So why exactly does something like`add2virtualenv "$(python dev_tools/modules.py list --mode package-path | xargs)"` not work ?

Reply:
Because add2virtualenv is a shell function.  Depending on how this shell function was installed, one can do xargs bash -c "add2virtualenv ..."  But this doesn't always work depending on how virtualenvwrapper.sh is set up. 

The alternative here is to add a command to dev_tools to change the path, which I think is doable.  Want this @MichaelBroughton ?

Reply:
Ahhhh in that case I'm not too bothered. I think adding an extra command is totally fine. I don't think this little sequence here warrants its own dev_tools command.